misc/docerina/src/main/java/org/ballerinalang/docgen/docs/BallerinaDocConstants.java

```diff
@@ -34,7 +34,8 @@
     public static final String ORG_NAME = "org-name";
     public static final String VERSION = "version";
     public static final String RECORD_TEMPLATE_NAME_KEY = "record.template.name";
-    public static final String OBJECT_TEMPLATE_NAME_KEY = "object.template.name";
+    public static final String CLASS_TEMPLATE_NAME_KEY = "class.template.name";
```

Comment:
Access can be package-private for both consts

Reply:
Added the changes